src/modules/classAwesome.js

```diff
@@ -0,0 +1,63 @@
+export default class Awesome {
+  constructor() {
+    this.data = {};
+    this.record = [];
+  }
+
+  addRecord(chore) {
+    this.data = { chore };
+    this.record.push(this.data);
+    return this.data;
+  }
```

Comment:
- [ ] each new item should have an "index" attribute and "completed" attribute. the completed attribute should default to false.